sqs/src/main/scala/akka/stream/alpakka/sqs/SqsModel.scala

```diff
@@ -117,96 +167,89 @@ object MessageAction {
 
 /**
  * Messages returned by a SqsFlow.
- * @param metadata metadata with AWS response details.
- * @param message message body.
+ * @param message the SQS message.
  */
-final class SqsPublishResult private (val metadata: com.amazonaws.services.sqs.model.SendMessageResult,
-                                      val message: String) {
-
-  /** Java API */
-  def getMetadata: com.amazonaws.services.sqs.model.SendMessageResult = metadata
-
-  /** Java API */
-  def getMessage: String = message
+final class SqsPublishResult private (
+    val metadata: AmazonWebServiceResult[ResponseMetadata],
+    val message: Message,
+    val fifoMessageIdentifiers: Option[FifoMessageIdentifiers]
```

Comment:
Add a `def getFifoMessageIdentifiers: Optional[FifoMessageIdentifiers]` as Java API, please.

Reply:
Ok. I'll write the Java accessors for all models